whist/server/web_socket/subscriber.py

```diff
@@ -23,5 +25,5 @@ def send(self, event: Event) -> None:
         :param event: Any type of event.
         :return: None
         """
-        wrapped_event = {'name': event.name, 'event': event}
-        self._connection.send_json(wrapped_event)
+        wrapped_event = {'name': event.name, 'event': event.json()}
+        asyncio.run(self._connection.send_json(wrapped_event))
```

Comment:
why use asyncio here when this is already happening in a background task?
asyncio.run is also really wasteful and should only ever be used to start one event loop in the beginning.

Reply:
Yes right, forgot to remove